src/paper/related_models/paper_model.py

```diff
@@ -392,111 +378,107 @@ def calculate_hot_score(paper):
             boost_time__gte=0,
         )
         today = datetime.datetime.now(tz=pytz.utc).replace(hour=0, minute=0, second=0)
-        score = paper.paper_score
-        unified_doc = paper.unified_document
+        score = paper.score
```

Comment:
@yattias Unrelated to this specific line but rather the method itself - are we currently using this to calculate hotscore for unified docs? I don't think so, so we should just delete this function @calvinhlee23

Reply:
Looks like respective documents have their own `calculcate hotscore` function that is being called at UnifiedDocument level. I'll keep this for now.